src/email_requests/urls.py

```diff
@@ -4,5 +4,6 @@
 
 
 urlpatterns = [
-    url(r'^(?P<mentor_id>[0-9]+)/$', views.EmailRequestView.as_view(), name='send_email')
```

Comment:
this is why I like trailing commas.  Git shows that you changed this line, but you really didn't